### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by satriamotorsport**
testing pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # eLibApp
 Self learning on NodeJS ExpressJS and MongoDB
+satriax was here
```
Comment (satriax) [inline README.md]: ga jelas

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # eLibApp
 Self learning on NodeJS ExpressJS and MongoDB
```
Comment (satriax) [inline README.md]: the original content

File: README.md (hunk 1 above)
Comment (satriamotorsport) [reply README.md]: masak sih?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # eLibApp
```
Comment (satriax) [inline README.md]: Add description

Comment (satriamotorsport) [reply README.md]: oke bos

Comment (satriax) [inline README.md]: sudah ok

Comment (satriax) [inline README.md]: done

**Pull request merged**